html/index.html

```diff
@@ -0,0 +1,207 @@
+
+	<!-- Twitter Cards -->
+	<meta name="twitter:card" content="summary">
+	<meta name="twitter:site" content="https://bugmark.net">
+	<meta name="twitter:title" content="Bugmark">
+	<meta name="twitter:description" content="Bugmark is the fastest way to build issue futures into your App.">
+
+	<!-- Facebook Open Graph -->
+	<meta property="og:locale" content="en_US">
+	<meta property="og:url" content="https://bugmark.net">
+	<meta property="og:site_name" content="Bugmark">
+	<meta property="og:title" content="Bugmark">
+	<meta property="og:description" content="Bugmark is the fastest way to build issue futures into your App.">
+	<meta property="og:type" content="website">
+
+	<!-- Title -->
+	<title>Bugmark</title>
+
+	<!-- Fonts -->
+	<link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
+
+	<!-- CSS -->
+	<link rel="stylesheet" href="assets/vendor/bootstrap/dist/css/bootstrap.min.css">
+</head>
+
+<body>
+	<!-- navigation -->
+	<nav class="navbar navbar-light">
+		<div class="container">
+			<a class="navbar-brand h1" href="#">Bugmark</a>
+			<button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+				<span class="navbar-toggler-icon"></span>
+			</button>
+			<div class="collapse navbar-collapse show" id="navbarNav">
+				<ul class="nav">
+					<li class="nav-item"><a class="nav-link" href="#">App</a></li>
+					<li class="nav-item"><a class="nav-link" href="#">Blog</a></li>
+					<li class="nav-item"><a class="nav-link" href="#">Docs</a></li>
+					<li class="nav-item"><a class="nav-link" href="#">Development</a></li>
+				</ul>
+			</div>
+		</div>
+	</nav>
+	<!-- //navigation -->
+	<!-- main -->
+	<main role="main">
+		<!-- hero -->
+		<section id="hero" class="jumbotron jumbotron-fluid mb-0 text-center">
+			<div class="container">
+				<h1 class="display-4 pt-5 pb-5">The world's fastest way to integrate issue futures into your app</h1>
+				<a class="btn btn-lg btn-outline-primary" href="">Bugmark.net</a>
+				<p class="lead mb-0 pt-5">Discover which issues really matter.</p>
+				<p class="lead">Directly pay developers who fix them.</p>
+			</div>
+		</section>
+		<!-- //hero -->
+		<!-- about -->
+		<section id="about" class="pt-5 pb-5">
+			<div class="container">
+				<div class="section-heading pt-5 pb-5 text-center">
+					<h3 class="display-4">What is Bugmark?</h3>
+				</div>
+				<div class="section-content pb-5">
+					<p>An open-source blockchain market platform to conduct research into the use of incentive algorithms and prediction markets as tooling for community managed software vulnerabilities and issues. <a href="https://mvscorg.github.io/bugmark-design/about/">Find more about Bugmark</a>.</p>
+				</div>
+			</div>
+		</section>
+		<!-- //about -->
+		<!-- how -->
```

Comment:
Instead of having to click multiple times to get the desired information, we could directly show the information on how the application works for developers and investors and project managers.